### DISCUSSION TITLE
Hope for official rust bindings!

### DISCUSSION THREAD

**Discussion by SethWen** [libvips/libvips#3051] [Help wanted]
I tried other similar-feature rust crate, but the performance is  even worse than nodejs-bindings！
Hope for reply. 
Thanks~

**Comment by jcupitt** [comment]
Hi @SethWen, I like Rust and I'd be interested in making a binding, but it's a question of finding a couple of weeks free :( It's not going to happen any time soon.

**Reply by jcupitt** [reply]
(I'd be very happy to advise and review code on any binding projects run by someone else, of course)
